auth/src/main/java/com/firebase/ui/auth/provider/TwitterProvider.java

```diff
@@ -92,9 +92,27 @@ public void startLogin(Activity activity) {
     }
 
     @Override
-    public void success(Result<TwitterSession> result) {
-        mTwitterAuthClient.requestEmail(result.data,
-                new EmailCallback(result.data, mCallbackObject));
+    public void success(final Result<TwitterSession> sessionResult) {
+        TwitterCore.getInstance()
+                .getApiClient()
+                .getAccountService()
+                .verifyCredentials(false, false, true)
```

Comment:
TBH, I just copied that from StackOverflow 😆. I did a bit of debugging and figured some of it out (in order): `includeEntities` will populate the `User#entities` field but we don't need it, `skipStatus` I have no idea what that does, `includeEmail` includes the email (duh! 😄).

Do you want me to document the parameter names? Intellij shows them, but it could be useful for GitHub:
![image](https://user-images.githubusercontent.com/9490724/28385057-91d21218-6c7b-11e7-843b-43c221cdd307.png)